lms/exceptions.py

```diff
@@ -13,7 +13,19 @@ class LTILaunchError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
     pass
 
 
+class LTIContentItemError(HTTPBadRequest):  # pylint: disable=too-many-ancestors
+    """Base exception for problems handling LTI content item selection."""
+
+    pass
+
+
 class MissingLTILaunchParamError(LTILaunchError):  # pylint: disable=too-many-ancestors
     """Exception raised if params required for lti launch are missing."""
 
     pass
+
+
+class MissingLTIContentItemParamError(LTIContentItemError):  # pylint: disable=too-many-ancestors
```

Comment:
I believe this could inherit from `LTILaunchError` as, again, "content selection" is a kind of launch. Then you won't need the `LTIContentItemError` class.